app/assets/javascripts/discourse/tests/acceptance/admin-user-index-test.js

```diff
@@ -63,6 +63,22 @@ acceptance("Admin - User Index", function (needs) {
     );
   });
 
+  test("can view post edits", async function (assert) {
+    await visit("/admin/users/1/eviltrout");
+    assert.equal(queryAll(".post-edits-modal").length, 0);
+
+    assert.equal(
+      queryAll(".display-row.post-edits-count .value").text().trim(),
+      "6\n     View Edits"
```

Comment:
It would be more future proof to make this two tests, one for each field. You might need to put one in a `<span>` or something to target it directly with a `querySelector` but it's worth it!